### PR TITLE
TVB-2875: Define a Jenkins job to test migration from 1.5.8

### PR DESCRIPTION
**Pull request by VinczeRobert**

### CODE REVIEW

File: tvb_build/docker/Dockerfile-migrate [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM tvb-run
```
Comment (bvalean) [inline tvb_build/docker/Dockerfile-migrate:1]: Do you want to use the **thevirtualbrain/tvb-run** image?

Comment (VinczeRobert) [reply tvb_build/docker/Dockerfile-migrate:1]: Yes

Comment (bvalean) [reply tvb_build/docker/Dockerfile-migrate:1]: Then you have to use full image name thevirtualbrain/tvb-run

File: tvb_build/Jenkinsfile-Migrate [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                }
+            }
+            steps {
+                sh '''#!/bin/bash
+                    source activate tvb-run
+                    cd tvb_build
+                    bash install_full_tvb.sh
+                    cd ..
+                    python -m tvb.config.init.migrated_files_counter
+                    xCopy /home/tvb_user/TVB_STORAGE/logs . /E/I
```
Comment (popaula937) [inline tvb_build/Jenkinsfile-Migrate:22]: maybe use `cp -R` command for the copy step

File: tvb_build/docker/Dockerfile-migrate (hunk 1 above)
Comment (popaula937) [reply tvb_build/docker/Dockerfile-migrate:1]: The image is currently built on Robert's local machine, not on Jenkins. As a next step, we will try to build it on Jenkins and use the existing thevirtualbrain/tvb-run, but then we will need to keep the generated project in a central place.

**Review by popaula937 — APPROVED** [review]

**Pull request merged**